README.md

```diff
@@ -106,22 +108,86 @@ The deployment settings such as container image and replicas can be changed by e
 kubectl kustomize deployments/kubernetes/dev --enable-helm | kubectl apply -f -
 ```
 
-### Simple Test
-To do a simple test. Please use the below tool as follows
+### [Kind](https://kind.sigs.k8s.io): Simple Test
+
+This test will bring up Merak and [Alcor](https://github.com/futurewei-cloud/alcor) in a
+single master node [Kind](https://kind.sigs.k8s.io) Kubernetes cluster.
+
+#### Prerequisites
+
+- Minimum Machine Requirements (Our tests were ran on AWS t2.2xlarge ec2 instances)
+  - 16GB RAM
+  - 8 Core CPU
+
+- [Make](https://www.gnu.org/software/make/)
+```
+sudo apt-get install make
+```
+- [Helm](https://helm.sh/docs/intro/install/)
+```
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+```
+- [Linkerd CLI](https://linkerd.io/2.12/getting-started/)
+```
+curl --proto '=https' --tlsv1.2 -sSfL https://run.linkerd.io/install | sh
+export PATH=$PATH:/home/ubuntu/.linkerd2/bin
+```
+- [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+```
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.17.0/kind-linux-amd64 && chmod +x ./kind && sudo mv ./kind /usr/local/bin/kind
+```
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+```
+curl -LO https://dl.k8s.io/release/v1.26.0/bin/linux/amd64/kubectl && sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+- [Docker](https://www.docker.com)
+```
+sudo apt-get install docker.io
+```
+  - Add current user to docker group (for running docker without sudo)
+```
+sudo groupadd docker
+sudo gpasswd -a $USER docker
+newgrp docker
+```
+
+#### Step 1: Deploy
+
+You can deploy Merak and [Alcor](https://github.com/futurewei-cloud/alcor) in [Kind](https://kind.sigs.k8s.io) with the command below.
+
+```
+git clone https://github.com/futurewei-cloud/merak.git
+cd merak
+make kind
+```
+
+Please wait for all pods to be in running state as shown in the picture below
+before proceeding to the next step. This should take approximately 5 minutes.
+
+![Successful Merak Deployment](docs/images/merak_successful_deployment.jpg)
+
+#### Step 2: Run test
+
+You can use the prebuilt test tool as shown below.
 
 ```
 ./tools/teste2e/bin/teste2e
 ```
```

Comment:
Don't forget to mention 10 :-)

Reply:
I've fixed this!

Reply:
oh, did you push to the PR? 

https://github.com/phudtran/merak/tree/deployment-optimize#step-2-run-test still shows no parameter passing.

Reply:
Yes the binary has been updated. Users will not need to pass in command line arguments

Reply:
aha, that was the trick. can we keep the binary while allowing user to pass in the number of vhost to be started, which is more configurable.

Reply:
I think we can think about expanding this teste2e tool in a future release (Configure VPC, Subnet, sg, vhost evms, etc. For now I'd like to keep it as is with no config from the user as changing the number of vhosts also affects the way the evms are scheduled.

Reply:
Okay sounds good to me.